tests/test0185.d

```diff
@@ -0,0 +1,22 @@
+//T compiles:yes
+//T retval:0
+//T has-passed:yes
+// Tests nested switch 
+int main() {
```

Comment:
sorry for indentation problem but my editor has 4 space = 1 tab setting and I think SDC code is using 8 space = 1 tab. I will do that change. How these test cases can be combined into one ? because 3 of them should not compile and one should.